extensions/wikia/AdEngine/js/EvolveHelper.js

```diff
@@ -56,9 +56,10 @@ define('ext.wikia.adEngine.evolveHelper', [
 	}
 
 	function getKruxKeyValues() {
-		if (Krux && Krux.dartKeyValues) {
-			return dartUrl.trimParam(Krux.dartKeyValues);
+		if (krux) {
+			return dartUrl.decorateParam('segments', krux.getParams('segments'));
```

Comment:
Makes sense to have krux.getSegments to return limited number of segments and sharing for consistency with GPT (see I contradict myself here? :D)